src/Room.js

```diff
@@ -53,7 +53,7 @@ class Rooms extends Bookings {
   calculateTotalRevenue(date) {
     let totalBill = 0;
     this.getTodaysBookings(date).forEach(booking => {
-      let room = this.rooms.find(room => room.number === booking.roomNumber)
+      let room = this.rooms.find(room => room.number === parseInt(booking.roomNumber))
```

Comment:
there are a few bookings with roomNumbers as strings parseInt fixes that problem